Updated the troubleshooting guide according to Guy's comments (#646)
run-ai/docs: docs/admin/troubleshooting/cluster-health-check.md

    `kubectl get pods -n runai | grep -E 'runai-agent|cluster-sync|assets-sync'`

>    If one or more of the services are not running, use the guidelines described under [Cluster has Service issues](#cluster-has-service-issues).

Reviewer: @gal-revach 
```suggestion
   If one or more of the services are not running, see [Cluster has Service issues](#cluster-has-service-issues).
```

Author: @jasonnovichRunAI looks good

    ```

    Replace `control-plane-endpoint` with the URL of the Control Plane in your environment.

    If the pod has failed to connect to the Control Plane, check for potential network issues. Use the following guidelines:
  
    * Ensure that the network policies in your Kubernetes cluster allow communication between the Control Plane and the Run:ai services that communicate with the Control Plane.
    * Check both Kubernetes Network Policies and any network-related configurations at the infrastructure level.
    * Verify that the required ports and protocols are not blocked.

3. Verify Run:ai services logs. Inspecting the logs of the Run:ai services that communicate with the CP is essential to identify any error messages.

    Run the following command on each one of the services:

    ```bash
    kubectl logs deployment/runai-agent -n runai
    kubectl logs deployment/cluster-sync -n runai
    kubectl logs deployment/assets-sync -n runai
    ```

>     From the logs, try to identify the problem. If you cannot resolve the issue, continue to the next step. 

Reviewer: @gal-revach 
```suggestion
   Try to identify the problem from the logs. If you cannot resolve the issue, continue to the next step. 
```

> 4. If the issue persists after completing the previous steps, contact Run:ai support for assistance.

Reviewer: @gal-revach 
```suggestion
4. If the issue persists, contact Run:ai support for assistance.
```
